package.json

```diff
@@ -25,25 +25,22 @@
   "homepage": "https://github.com/backtrace-labs/backtrace-morgue#readme",
   "dependencies": {
     "@backtrace/node": "^0.3.1",
+    "@cypress/request": "^3.0.8",
```

Comment:
types definition shouldnt be in dependencies.

Reply:
Oh, wait, I misread this as "type definitions _should_ be in dependencies". This `@cypress/request` package is a maintained fork of `request` (switched to the former as part of fixing `npm audit` issues).

Reply:
I don't see any reason that this couldn't be used outside of Cypress.
We should still strive to replace it with `axios`, perhaps this will be easier once we fully migrate to TS.